webpack.staticData.js

```diff
@@ -0,0 +1,19 @@
+const webpack = require("webpack");
+const publisherConfig = require("./webpack.publisher");
+
+
+let staticData = publisherConfig;
+
+staticData.publisherConfig.mode = "none";
+staticData.publisherConfig.plugins.push(new webpack.DefinePlugin({
+    "process.env.NODE_ENV": JSON.stringify("staticData")
```

Comment:
it could be better to have const definitions for environments because you use this static string in another file as well. so that it could be a single source, easy to read, and fault-tolerant.

Reply:
It needs to be escaped as that's how webpack checks it:
![image](https://user-images.githubusercontent.com/92857141/154043440-5c4a3a67-1f52-49de-9822-74011b0e0276.png)